### DISCUSSION TITLE
How to stop miner?

### DISCUSSION THREAD

**Discussion by ghost** [Lolliedieb/lolMiner-releases#382] [Q&A]
total noob here, i clicked on the mine_eth.bat file and everythings going great, but i dont know how to stop the miner

**Comment by ChesDonno** [comment]
crtl+c

**Reply by Christoffer2020** [reply]
Is that totally safe? Isn't there an 'exit' command or something?

**Reply by Shillelagh-Hummus** [reply]
Ctrl+C is how you normally exit batch files and other executables in windows. Programmers can even write statements to run before the program is shut down using ctrl+c so it can save everything, and in this cas ask you to shut down the mining operation. It's totally safe.

**Reply by Rhacknar** [reply]
i litteraly just hit the x in top right.... never have an issue with that
